### PR TITLE
Collect new snapshots

### PR DESCRIPTION
**Pull request by justinjeffery-ipf**
Nautobot still was not automatically refreshing snapshots.  This should fix the issue.

### CODE REVIEW

**Comment by justinjeffery-ipf** [comment]
Tested and now anytime someone goes to the page to start a sync it will now collect new snapshots like it should.

**Comment by justinjeffery-ipf** [comment]
Any idea how we can correctly do this so it updates snapshots once instead of twice?  Another customer having issues not viewing recent snapshots and only seeing old data.

File: nautobot_ssot_ipfabric/jobs.py [hunk 1]
```diff
@@ -219,6 +225,8 @@ def load_target_adapter(self):
 
     def sync_data(self):
         """Sync a device data from IP Fabric into Nautobot."""
+        if self.client is None:
+            self.client = self._init_ipf_client()
```
Comment (pke11y) [inline nautobot_ssot_ipfabric/jobs.py:229]: could perhaps just refresh the client without checking, but no big deal.

**Review by pke11y — APPROVED** [review]
Tested and it works.

**Pull request merged**